### DISCUSSION TITLE
Replace GNOME Web (Epiphany) with another browser in future releases

### DISCUSSION THREAD

**Discussion by joeljosedev** [elementary/triage#114] [Ideas]
Even though Epiphany blends in with the rest of the Pantheon desktop in elementary OS and the other GNOME apps, it is not nearly as good as its counterparts (Brave and Firefox to name a few). So I think it's a good idea to replace Epiphany with another browser like Brave with a custom theme that makes it blend in with the rest of the desktop.

Here's a video of The Linux Experiment pointing out the drawbacks of Epiphany: https://youtu.be/VjQQhuVS26g

**Comment by danirabbit** [comment]
Hey thanks for the suggestion! At the moment there aren’t any viable alternatives to Epiphany since we only ship GTK apps in elementary OS and Epiphany is the most feature complete and well maintained GTK web browser available.

since this post was made there have been really big improvements in speed and web standards compatibility in Epiphany, so I’m going to close as “outdated”
